lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2937,46 +2937,57 @@ namespace {
       if (!clangDecl->hasAttrs())
         return;
 
-      SmallVector<ValueDecl *, 1> results;
-      auto conformsToAttr =
-          llvm::find_if(clangDecl->getAttrs(), [](auto *attr) {
-            if (auto swiftAttr = dyn_cast<clang::SwiftAttrAttr>(attr))
-              return swiftAttr->getAttribute().starts_with("conforms_to:");
-            return false;
-          });
-      if (conformsToAttr == clangDecl->getAttrs().end())
+      SmallVector<clang::Attr*, 40> conformsToMatches;
+      llvm::copy_if(clangDecl->getAttrs(), std::back_inserter(conformsToMatches), [](auto *attr) -> bool {
+        if (auto swiftAttr = dyn_cast<clang::SwiftAttrAttr>(attr))
+          return swiftAttr->getAttribute().starts_with("conforms_to:");
+        return false; 
+      });
+
+      if (conformsToMatches.empty())
```

Comment:
The for loop below would run 0 times if this vector is empty. We might be able to remove this early return.